wrapper.js

```diff
@@ -6,71 +6,78 @@ var https = require('https');
 var MemoryStream = require('memorystream');
 
 function setupMethods (soljson) {
-  var compileJSON = soljson.cwrap('compileJSON', 'string', ['string', 'number']);
-  var compileJSONMulti = null;
-  if ('_compileJSONMulti' in soljson) {
-    compileJSONMulti = soljson.cwrap('compileJSONMulti', 'string', ['string', 'number']);
-  }
-  var compileJSONCallback = null;
-  var compileStandard = null;
-  if (('_compileJSONCallback' in soljson) || ('_compileStandard' in soljson) || ('_solidity_compile' in soljson)) {
```

Comment:
When did we use `_solidity_compile` and where is it now?

Reply:
It is the new API, present in 0.5.0.

Reply:
Added here https://github.com/ethereum/solidity/pull/4366

Reply:
ok, but where is the function wrapper generated after the PR?

Reply:
Line 78. It is an alias to `compileStandard`, so if it is found, takes precedence for `compileStandard`.

Reply:
Easier to review this PR commit-by-commit with whitespace changes disabled.